data/policies/filecon.cas

```diff
@@ -6,6 +6,7 @@ resource foo {
 	file_context("/etc/somesymlink", [lnk_file], this);
 	file_context("/var/somepipe", [fifo_file], this);
 	file_context("/var/somesocket", [sock_file], this);
+	file_context("/bin/some_bin", [file], system_u:object_r:foo:s0);
```

Comment:
From parser.lalrpop we are not using the mls right now, but it may be a good idea to add a file context with an mls range here as well to make sure we can parse that when its ready.

Reply:
Okay, very glad we decided to dig into this, because there's a problem with mls ranges.  The category portion of a label can be a comma-separated list.  So if we have something like:

```
some_call(system_u:object_r:type_t:s0-s0:c0,c1);
```

It's not clear if that's a function with one argument (`system_u:object_r:type_t:s0-s0:c0,c1`), or two arguments (`system_u:object_r:type_t:s0-s0:c0` and `c1`).

My first instinct is to say that we need to quote the full contexts or something, but that's a little annoying given that they contain elements declared elsewhere in the policy (eg types).

Reply:
Matt and I discussed this offline, and both think that allowing contexts to be optionally quoted, and requiring the quotes if they will contain commas, similar to how bash strings work with spaces is the most sensible approach here.

I've updated the PR to assume contexts can't contain commas, and the quote portion, along with helpful errors to point out that a comma in a context might need to be quoted will be future work.